debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/GCClassArrayClassSlotIterator.java

```diff
@@ -47,6 +47,11 @@ protected GCClassArrayClassSlotIterator(J9ClassPointer clazz) throws CorruptData
 			slots.add(slot);
 			addresses.add(VoidPointer.cast(clazz.arrayClassEA()));
 		}
+		slot = clazz.nullRestrictedArrayClass();
+		if(slot.notNull()) {
+			slots.add(slot);
+			addresses.add(VoidPointer.cast(clazz.nullRestrictedArrayClassEA()));
+		}
```

Comment:
Should we add `companionArray` as well?

Reply:
class.nullRestrictedArrayClass is the same object as class.arrayClass.companionArray